.cirrus.yml

```diff
@@ -257,22 +273,17 @@ task:
 
 task:
   name: '[ASan + LSan + UBSan + integer, no depends, USDT] [lunar]'
+  enable_bpfcc_script:
+    # In the image build step, no external environment variables are available,
+    # so any settings will need to be written to the settings env file:
+    - sed -i "s|\${CIRRUS_CI}|true|g" ./ci/test/00_setup_env_native_asan.sh
```

Comment:
Might it be more straightforward to define `BPFCC_PACKAGE` in `ci\test_imagefile`?

Running `FILE_ENV="./ci/test/00_setup_env_native_asan.sh" ./ci/test_run_all.sh` locally skips `interface_usdt_*.py` tests.

Reply:
oh yeah I see this locally too (on master), even though `--cap-add SYS_PTRACE` is in the docker command and `USDT tracing` is set by configure, what's missing?

Reply:
> what's missing?

These are the checks to determine if we want to skip the USDT test: https://github.com/bitcoin/bitcoin/blob/2fa60f0b683cefd7956273986dafe3bde00c98fd/test/functional/interface_usdt_net.py#L87-L91

I haven't looked into this in detail. With which user are the test run in docker locally? Is `--cap-add SYS_PTRACE` enough?

Reply:
When I set CIRRUS_CI to true locally I end up failing installing the kernel headers. I see this error running this branch locally on both Ubuntu and macOS hosts. But the "generic" version there is coming from inside the container right (`uname --kernel-release`)? So I'm confused why this works on actual CI but not locally.

```
#8 7.217 E: Unable to locate package linux-headers-5.15.0-78-generic
#8 7.217 E: Couldn't find any package by glob 'linux-headers-5.15.0-78-generic'
#8 7.217 E: Couldn't find any package by regex 'linux-headers-5.15.0-78-generic'
```

Reply:
Ah 🤦‍♂️ the comment you added in response to my first review. Thanks!